composer.json

```diff
@@ -23,6 +23,7 @@
         "symfony/validator": "~2.3||~3.0.0||~3.1.0||^3.2.4||~4.0",
         "symfony/filesystem": "~2.3||~3.0||~4.0",
         "symfony/config": "~2.3||~3.0||~4.0",
+        "symfony/dotenv": "~2.3||~3.0||~4.0",
```

Comment:
Instead of a hard dependency, this could also be a soft one, and we check if the class exists or not.
This way, most users not needing this wont require an additional dependency here.